arwen/elrondapi/elrondei.go

```diff
@@ -506,6 +507,10 @@ func getNumArguments(context unsafe.Pointer) int32 {
 	return int32(len(args))
 }
 
+func isElrondReservedKey(key []byte) bool {
```

Comment:
Move as a public method of `storageContext`.

Reply:
+1

Reply:
+1